tests/curious_tests/models.py

```diff
@@ -27,6 +27,15 @@ def authors(instances, filters=None):
     return r
 
 
+class UserTest(models.Model):
```

Comment:
Not sure why this is called user here. Another noun from the existing world of authors, etc might be clearer. Or an explanation of why User is meaningful

Reply:
I couldn't figure out a legit One-To-One name for Authors last night. The canonical case is for extending the user object so I just made it User

Reply:
Author can now be a Person